src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js

```diff
@@ -383,6 +388,21 @@
             return arr;
         }
 
+        function setSelectedFilter() {
+            var filterParameters = filterParametersService.getSelectedFilter()[vm.dimension.name]
+            if (filterParameters) {
+                if (vm.list[vm.dimension.name] && vm.list[vm.dimension.name]) {
+                    var selectedFilter = vm.list[vm.dimension.name].filter(function (item) {
+                        return filterParameters.indexOf(item) !== -1
```

Comment:
What is the use of return as you are only setting the list?

Reply:
return is part of the filter  option